Unaccountable-Magpie/BangazonAPI2
# Ag ordercontroller1

Purpose:
Allow developers to access the Order resource #4

How it fits in context of the project:
Accomplishes part of the 


Specific Feature affected:
Adds the methods used to Get, Post, Put, and Delete.


How to test (Be thorough!):
(Shout out to Amanda for this awesome postman walk thru)


Checkout to my branch ag-ordercontroller1

open OrdersController

Press IIS Express play button to start up the server, a window will open inside the browser. You will need to add /api/orders to end of the url in order to access my controller. Copy this address as you will need to paste inside Postman

Open PostMan, and select GET

Copy that address from the browser inside there and press SEND

Select POST within PostMan

add a new JSON object, click on Body, select Raw, and change Text to JSON
ex. {     {
        "customersId": 2,
        "paymentTypesId": 2,
    }}

Rerun the GET and you should see a new object

Select PUT within PostMan

Update one of the JSON objects, click on Body, select Raw, and change Text to JSON
ex. {     {
        "customersId": 2,
        "paymentTypesId": 1,
    }}

Rerun the GET and you should see the updated object

Select DELETE within PostMan

Delete one of the JSON objects, click on Body, select Raw, and change Text to JSON
ex. {     {
        "id": 2,
        "customersId": 2,
    }"}

Rerun the GET and you shouldn't see that object because it was deleted
